src/app/page.jsx

```diff
@@ -0,0 +1,29 @@
+'use client'
+
+import { useEffect, useState } from 'react'
+import Message from '@/components/message'
+
+export default function Home() {
+  const [timeStamp, setTimeStamp] = useState(0)
+
+  useEffect(() => {
+    setTimeStamp(Date.now())
+  }, [])
+
+  return (
+    <main className="flex flex-col gap-3 p-3">
+      <Message sentAt={timeStamp}>
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quae
+        architecto quis iure ex id voluptatum reprehenderit quod cum laboriosam.
+        Dicta temporibus quis sed assumenda tempore perferendis voluptas magnam
+        minima dignissimos! Dignissimos porro ea incidunt quae blanditiis,
+        exercitationem deserunt beatae omnis temporibus officiis, consequatur
+        nulla accusamus commodi officia minus, ullam unde optio nisi similique
+        ab maiores? Perferendis aliquid minus laudantium error.
+      </Message>
+      <Message sentAt={timeStamp}>Hello, i'm a message! 😎</Message>
+      <Message sentAt={timeStamp}>I'm not empty</Message>
+      <Message sentAt={timeStamp}>...</Message>
+    </main>
+  )
+}
```

Comment:
I think when you declare `MOCK_MESSAGES` separately it looks better.

```suggestion
'use client'

import Message from '@/components/message'

const now = Date.now()
const MOCK_MESSAGES = [
  { sentAt: now - 1000 * 60 * 30, content: `...` },
  { sentAt: now - 1000 * 60 * 20, content: `I\'m not empty` },
  { sentAt: now - 1000 * 60 * 10, content: `Hello, i\'m a message! 😎` },
  {
    sentAt: now,
    content:
      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Id, porro',
  },
]

export default function Home() {
  return (
    <main className="flex flex-col gap-3 p-3">
      {MOCK_MESSAGES.map(({ sentAt, content }, i) => (
        <Message key={i} sentAt={sentAt}>
          {content}
        </Message>
      ))}
    </main>
  )
}
```

Reply:
Yeah, forgot about that. How long I haven't worked with React)